packages/wdio-cli/src/utils.ts

```diff
@@ -531,42 +533,39 @@ export async function getAnswers(yes: boolean): Promise<Questionnair> {
 
     const projectProps = await getProjectProps(process.cwd())
     const isProjectExisting = Boolean(projectProps)
+    const nameInPackageJsonIsNotCreateWdioDefault = projectProps?.packageJson?.name !== 'my-new-project'
```

Comment:
Where does `my-new-project` come from?

Reply:
@christian-bromann I didn't introduce this behavior, just refactored. Looking through git logs shows that this name [was used by `create-wdio` generator](https://github.com/webdriverio/create-wdio/blob/76a54560b1b71d3b8ebb46270605deb5b28d06c2/src/index.ts#L66-L70) and it got removed in [create-wdio@c3e5fd29b72](https://github.com/webdriverio/create-wdio/commit/c3e5fd29b72675a33701895a95736235295c35b3#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80L69)